packages/esm-patient-chart-app/src/root.scss

```diff
@@ -18,9 +19,23 @@
     width: 0px;
     background-color: transparent
   }
+
   :global(.cds--side-nav__overlay-active) {
     background-color: transparent;
   }
+
+  :global(.cds--side-nav) {
+    overflow: auto;
+  }
```

Comment:
Please take this code in the esm-styleguide, this is something needed for the whole app, not only for the patient chart